src/SceneGraphDebugCommandController.ts

```diff
@@ -0,0 +1,291 @@
+// eslint-disable-next-line
+const Telnet = require('telnet-client');
+
+export class SceneGraphDebugCommandController {
+    constructor(public host: string) {
+    }
+
+    private connection: typeof Telnet;
+    private connectionSuccessful = false;
```

Comment:
Can we eliminate this property in favor of checking `this.connection !== null`?  or if the connection has a status property, then something like `this.connection?.connectionState === 'connected'`?

Reply:
Yea, I think so. I'll just have to make sure I clear the `this.connection` on close and not set it till it's successful.